details.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
```

Comment:
Do you need this import for the code below? You might be able to eliminate it, and still run the function below properly

Reply:
I think you're right!